test/trento/clusters_test.exs

```diff
@@ -300,62 +347,62 @@ defmodule Trento.ClustersTest do
       assert :ok = Clusters.request_checks_execution(cluster_id)
     end
 
-    test "should start a checks execution on demand for ascs_ers clusters with a SAP system ENSA 1 version" do
-      sap_systems = build_list(1, :ascs_ers_cluster_sap_system, filesystem_resource_based: false)
-      [%Trento.Clusters.ValueObjects.AscsErsClusterSapSystem{sid: sid}] = sap_systems
+    test "should start a checks execution on demand for ascs_ers clusters with ENSA 1 version" do
+      %SapSystemReadModel{id: sap_system_id_1, sid: sid_1} =
+        insert(:sap_system, ensa_version: Trento.SapSystems.Enums.EnsaVersion.ensa1())
```

Comment:
You should use the aliased short version here `EnsaVersion.ensa1()` (and in many more places)

Reply:
This is because these tests use both `Trento.SapSystems.Enums.EnsaVersion` and `Trento.Clusters.Enums.EnsaVersion`. I'm fine with renaming the cluster's ENSA version something else like `ClusterEnsaVersion` or `AggregatedEnsaVersion`, what do you think?